README.md

```diff
@@ -1,35 +1,59 @@
 
 # Linting for technical writing
+
 This package contains rules based on the [GOV.UK Technical Style guide](https://www.gov.uk/guidance/style-guide/technical-content-a-to-z) and is designed to work with the linting tool Vale. You can [find more information about Vale on its website](https://vale.sh/).
 
 Errors raised by the linter will show for each file:
+
 - the line and character number of the issue
 - the severity level of the issue (error, warning, suggestion)
 - a description of the issue
 - the path to the rule that flagged the issue
 
+If you would like to suggest a change an existing rule, such as adding an acronym to our list of exceptions, please raise this in the [#ask-di-tech-writing-architecture](https://gds.slack.com/archives/C06V5UTTJNP) slack channel, and it will be discussed in our regular forum.
```

Comment:
```suggestion
If you would like to suggest a change an existing rule, such as adding an acronym to our list of exceptions, please raise this in the [#ask-di-tech-writing-architecture](https://gds.slack.com/archives/C06V5UTTJNP) Slack channel, and it will be discussed in our regular forum.
```